go.mod

```diff
@@ -34,8 +39,15 @@ require (
 	gotest.tools v2.2.0+incompatible
 	istio.io/api v0.0.0-20200227213531-891bf31f3c32
 	istio.io/istio v0.0.0-20200304114959-c3c353285578
+	rsc.io/letsencrypt v0.0.3 // indirect
 )
 
+replace github.com/Azure/go-autorest/autorest => github.com/Azure/go-autorest/autorest v0.11.15
+
+replace github.com/docker/docker => github.com/docker/docker v17.12.1-ce+incompatible
+
+replace github.com/hashicorp/consul => github.com/hashicorp/consul v1.3.1
```

Comment:
just curious, what are the purpose of these replacements?

Reply:
These replacements are mainly for resolving the dependency issues caused by other packages.